src/Behat/Behat/Context/Annotation/DocBlockHelper.php

```diff
@@ -24,6 +24,9 @@ public function extractDescription(string $docBlock): string
         // Remove indentation
         $description = preg_replace('/^[\s\t]*/m', '', $docBlock);
 
+        // normalize line endings
```

Comment:
While running the tests in CI I discovered that the actual output was different in Windows machines. This had been hidden by the manipulation of line endings that we do on the tests but when using fixtures it became noticeable. The issue is that we later split this description into lines by a newline character, so we need to normalize the line endings in Windows in order for this functionality to behave equally in all OSs